plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseSharedMetastoreTest.java

```diff
@@ -171,33 +171,37 @@ public void testTimeTravelWithRedirection()
     @Test
     public void testMigrateTable()
     {
+        String schemaName = "test_migrate_" + randomNameSuffix();
         String tableName = "test_migrate_" + randomNameSuffix();
-        String hiveTableName = "hive.%s.%s".formatted(schema, tableName);
-        String icebergTableName = "iceberg.%s.%s".formatted(schema, tableName);
+        String hiveTableName = "hive.%s.%s".formatted(schemaName, tableName);
+        String icebergTableName = "iceberg.%s.%s".formatted(schemaName, tableName);
 
+        assertUpdate("CREATE SCHEMA " + schemaName);
```

Comment:
Maybe let's fix or isolate testShowTables instead, rather than require every test to create new schema.

testShowTables can be isolated at execution, or by creating a new schema in that test
or fixed by relaxing assertions

Reply:
Updated the tests to create 2 schemas in `createQueryRunner` because other tests (e.g. testReadInformationSchema, testShowSchemas) also depend on the existing schema.